File: Solutions/0171- Excel Sheet Column Number/Solution.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        answer = 0

        for i in range(0, len(columnTitle)):
            answer += (ord(columnTitle[i]) - ord('A') + 1) * pow(26, len(columnTitle) - i - 1)

        return answer

File: Solutions/0171- Excel Sheet Column Number/Solution.java
class Solution {
    public int titleToNumber(String columnTitle) {
        int answer = 0;

        for (int i = 0; i < columnTitle.length(); i++) {
            answer += (columnTitle.charAt(i) - 'A' + 1) * Math.pow(26,columnTitle.length() - i - 1);
        }

        return answer;
    }
}